scripts/generate-roar.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+if [[ -z "$1" ]]; then
+    echo >&2 "Usage: $0 [ 01 | .. | 25 ] [ gee | roar ]"
+    echo >&2
+    exit
+fi
+
+if [[ ! "$2" =~ ^[0-9][0-9]$ ]]; then
+    echo >&2 "Usage: $0 [ 01 | .. | 25 ]"
+    echo >&2
+    echo >&2 "Create skeleton file from template in ${BASE_PATH}q<xx>.rs"
+    exit
+fi
+
+DAY=$2
+
+BASE_SRC_PATH="src/${1}/"
+BASE_INPUT_PATH="input/${1}/"
+
+SAMPLE_1="${BASE_INPUT_PATH}/q${DAY}_p1_sample.txt"
+SAMPLE_2="${BASE_INPUT_PATH}/q${DAY}_p2_sample.txt"
+PUZZLE_INPUT="${BASE_INPUT_PATH}/q${DAY}_p1_input.txt"
+
+echo "Setting up input files at input/rtob/input ..."
+touch $SAMPLE_1
+touch $SAMPLE_2
+touch $PUZZLE_INPUT
+
+echo "Writing output to ${BASE_SRC_PATH}q${DAY}.rs ..."
+cat >"${BASE_SRC_PATH}q${DAY}.rs" <<EOF
+use crate::gee::parser::FileLines;
```

Comment:
This is fine if you want to use the parser I added, but maybe we should just move it up to the top level?